biz.aQute.repository/src/aQute/bnd/repository/maven/pom/provider/Traverser.java

```diff
@@ -206,8 +207,14 @@ private void parseResource(Archive archive, String parent) throws Exception {
 			}
 			addInformationCapability(rb, archive.toString(), parent);
 		} catch (Exception e) {
+			Throwable theException = e;
```

Comment:
See the methods for this in Exceptions, they provide solid support to get the original cause

Reply:
Makes sense. I will use the ```Exceptions.unrollCause``` method.